measure/src/main/scala/org/apache/griffin/measure/configuration/dqdefinition/DQConfig.scala

```diff
@@ -74,9 +74,9 @@ case class DQConfig(@JsonProperty("name") private val name: String,
   */
 @JsonInclude(Include.NON_NULL)
 case class DataSourceParam( @JsonProperty("name") private val name: String,
-                            @JsonProperty("baseline") private val baseline: Boolean,
```

Comment:
is that reordering really required?

Reply:
No, it's not. But keeping fields with default values as last elements in case classes is kind of "best practice" in scala. For example, scala compilation fails for the following snippet of code

```
case class Foo(a: Int, b: Int = 22, c: Double)

val foo = Foo(1, 3.3)
```

For this particular class we can avoid reordering. But I see more reasons to do it right and don't see any reasons to keep it as is (except we are scared to change any pice of code).

Reply:
LGTM